puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_nightly.groovy

```diff
@@ -0,0 +1,69 @@
+def packaging_branch = 'rpm/3.7'
+def source_server = 'http://ah.testing.ansible.com/nightlies/latest/'
```

Comment:
I'm not very comfortable with pulling instructions over plain text HTTP. Is it available over something secure?

Reply:
Not today, but we can ask Yanis.

Reply:
Please do.